src/meta_schedule/postproc/postproc.cc

```diff
@@ -59,6 +59,14 @@ Array<Postproc> Postproc::DefaultLLVM() {
   };
 }
 
+Array<Postproc> Postproc::DefaultVNNI() {
+  return Array<Postproc>{
+      Postproc::DisallowDynamicLoop(),   Postproc::RewriteParallelVectorizeUnroll(),
+      Postproc::RewriteReductionBlock(), Postproc::RewriteTensorize(/*vectorize_init_loop=*/true),
+      Postproc::RewriteLayout(),
```

Comment:
Is `RewriteLayout` useful here? Since VNNI expects the layout of the weight to be fixed.

Reply:
I think it's useful as a fallback when auto tensorization can't apply

Reply:
I'm cool with this as long as `RewriteLayout` doesn't break tensorization.